File: core-java-practice/gcr-codebase/programming-elements/VolumeOfCylinder.java
import java.util.*;

public class VolumeOfCylinder {
    public static void main(String[] args){
        Scanner sc =new Scanner(System.in); // creating scanner object
        double rad = sc.nextDouble(); // taking radius input
        double h = sc.nextDouble(); // taking height input
        double vol = Math.PI * rad * rad * h; // calculating volume
        System.out.println(vol); // printing the volume
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/VolumeOfCylinder.cs

class VolumeOfCylinder {
    static void Main(String[] args){
        double rad = Convert.ToDouble(Console.ReadLine()); // taking radius input
        double h = Convert.ToDouble(Console.ReadLine()); // taking height input
        double vol = 3.14 * rad * rad * h; // calculating volume
        Console.WriteLine(vol); // printing the volume
    }
}
